app/src/main/java/com/github/h3lp3rs/h3lp/database/Databases.kt

```diff
@@ -15,7 +15,7 @@ import com.github.h3lp3rs.h3lp.storage.Storages.Companion.storageOf
  * Enumeration of all useful databases in H3LP
  */
 enum class Databases {
-    PREFERENCES, EMERGENCIES, NEW_EMERGENCIES, MESSAGES, CONVERSATION_IDS, PRO_USERS, FORUM;
+    PREFERENCES, EMERGENCIES, NEW_EMERGENCIES, MESSAGES, CONVERSATION_IDS, PRO_USERS;

```

Comment:
I guess there's some specific reason why you removed the forum database but I don't really understand it.

Reply:
It shouldn't be accessible via the Databases enumeration!
If we call databaseOf(FORUM) somewhere in the code, we risk breaking stuff. The forum interface should be the only point of access.